README.md

```diff
@@ -110,17 +112,17 @@ If you a working a Applicaster plugin then add the following configuration to yo
 //Only for iOS/tvOS
 "extra_dependencies": [
   {
-    "react-native-broadcast-manager": ":path => './node_modules/react-native-broadcast-manager'"
+    "react-native-broadcast-manager": ":path => './node_modules/@applicaster/react-native-broadcast-manager'"
   }
 ],
 //For all platforms
 "npm_dependencies": [
-  "react-native-broadcast-manager@^0.2.4"
+  "@applicaster/react-native-broadcast-manager@^0.2.4"
```

Comment:
@snooky23 can we omit setting version explicitly in manifest files? It will take latest version in that case, right?